src/routes/committee/schedule/availability/[rehearsalId]/+page.ts

```diff
@@ -0,0 +1,36 @@
+import { db } from '$lib/firebase/client/firebase'
+import { collection, collectionGroup, doc, getDoc, getDocs, query, where } from 'firebase/firestore'
+import type { PageLoad } from './$types'
+import type { rehearsal } from '$lib/types/domain/rehearsal'
+import type { availability } from '$lib/types/domain/availability'
+import type { user } from '$lib/types/domain/user'
+
+export const load: PageLoad = async ({ params }) => {
+	const rehearsalRef = doc(db, 'rehearsals/', params.rehearsalId)
+	const rehearsal = (await getDoc(rehearsalRef)).data() as rehearsal
+
+	const usersQuery = collection(db, 'users')
+	const users = Object.assign(
+		{},
+		...(await getDocs(usersQuery)).docs.map((doc) => ({
+			[doc.id]: doc.data() as user
+		}))
+	)
+
+	const availabilityQuery = query(
+		collectionGroup(db, 'availability'),
+		where('rehearsal', '==', rehearsalRef)
+	)
+
+	const availabilitiesDict = Object.assign(
+		{},
+		...(await getDocs(availabilityQuery)).docs.map((document) => {
+			if (document.ref.parent.parent != null) {
+				return {
+					[document.ref.parent.parent.id]: document.data() as availability
+				}
+			}
+		})
+	)
```

Comment:
Weet je zeker dat dit goed gaat met meerdere availability per persoon? Want vgm krijg je er nu maar 1 pp terug omdat je de parent id override in de dict telkens als je er 1 tegen komt

Reply:
Oh wait, maar je hebt maar 1 availability per rehearsal zeker, dan kun je resolven I guess